### PR TITLE
Change from description based category description showing to js

### PR DESCRIPTION
**Pull request by marek-kramski**

### CODE REVIEW

File: dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
       coiHideDetails.style.display = 'block';
 
       if (!isMobile) {
-        document.getElementById('coi-consent-banner__category-expander-cookie_cat_necessary').checked = true;
```
Comment (dmazur) [inline dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html:18]: is that ok? we'll just change the color here?

**Review by dmazur — COMMENTED** [review]
I have a problem with the JS here. We're doing much manipulation for styles instead of keeping it in CSS and just changing the classes.

**Comment by marek-kramski** [comment]
I've made some changes - I moved styling from js to css. Now js is only toggling active class when necessary.

File: dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html (hunk 1 above)
Comment (marek-kramski) [reply dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html:18]: You left a comment on the file that was changed. It is now done in the way that you mentioned - by changing classes.

File: dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html [hunk 2]
```diff
@@ -46,13 +46,34 @@
...
-    var isMobile = window.matchMedia('(max-width: 760px)').matches;
-    var categoryCheckbox = document.getElementById(id);
+  var currentCategory = '';
+
+  function showDescription(id) {
+    var categoryNames = document.getElementsByClassName('coi-consent-banner__category-name');
+    var descriptions = document.getElementsByClassName('coi-consent-banner__description-container');
+    var category = document.getElementById(id);
+    var descriptionContainer = document.getElementById('coi-consent-banner__description-container-' + id);
+    var description = document.getElementById('description-' + id);
```
Comment (adamlukaszczyk) [inline dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html]: those ids should also have `coi` prefix

**Review by wojtek0207 — APPROVED** [review]

**Pull request merged**